xml/System.Text.Json.Nodes/JsonArray.xml

```diff
@@ -33,8 +33,17 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents a mutable JSON array.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+It is safe to perform multiple concurrent read operations on a <xref:System.Text.Json.Nodes.JsonArray>,
+            but issues can occur if the collection is modified while it's being read.
```

Comment:
```suggestion
It is safe to perform multiple concurrent read operations on a <xref:System.Text.Json.Nodes.JsonArray>, but issues can occur if the collection is modified while it's being read.
```

Reply:
Looks like a tooling bug as the code has
```
    /// <remarks>
    /// It is safe to perform multiple concurrent read operations on a <see cref="JsonArray"/>,
    /// but issues can occur if the collection is modified while it's being read.
    /// </remarks>
```
it added 12 spaces after the newline. however it comes out right:
![image](https://user-images.githubusercontent.com/6385855/128286553-b95a3257-dc2f-4195-84b5-625dd838df62.png)

@gewarren it seems the build process removes extra spaces/tabs within the text?

Reply:
Yes, extra whitespace is ignored.